sparse/tests/test_core.py

```diff
@@ -130,12 +131,43 @@ def test_dot():
     a = random_x((3, 4, 5))
     b = random_x((5, 6))
 
+    la = a.tolist()
+    lb = b.tolist()
+    la, lb          # silencing flake8
+
     sa = COO.from_numpy(a)
     sb = COO.from_numpy(b)
 
     assert_eq(a.dot(b), sa.dot(sb))
     assert_eq(np.dot(a, b), sparse.dot(sa, sb))
 
+    if sys.version_info >= (3, 5):
```

Comment:
Rather than an explicit version check, perhaps we could check ``hasattr(operator, 'matmul')``